addshare/meson.build

```diff
@@ -7,4 +7,5 @@ addshare = executable(
   include_directories: tools_incdir,
   link_with: libksmbdtools,
   install: true,
+  install_dir: get_option('prefix') / get_option('sbindir'),
```

Comment:
You can just use sbindir without prefix, because install_dir when given a relative path will install it relative to the prefix already.